src/sentry/assistant/manager.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+import six
+
+
+class AssistantManager(object):
+    def __init__(self):
+        self._id_registry = {}
```

Comment:
Let's rename this to `_guides` or similar since it's holding all the guides not just their ids